src/web/components/TeamMember/TeamMembersTable.scss

```diff
@@ -21,21 +21,28 @@
     }
 
     .email {
-      width: 60%;
+      width: 40%;
+    }
+
+    th.action {
+      text-align: center;
```

Comment:
This creates the same alignment problem I mentioned on the email contacts table.